packages/core/src/components/_index.scss

```diff
@@ -17,8 +16,11 @@
 @import 'StepList/StepList';
 @import 'Table/Table';
 @import 'Tabs/Tabs';
+// Mask and Date field styles need imported
+// after TextField since they modify it
```

Comment:
Nice find! This seems a little fragile, but I'm not sure what the best solution is right now. Right now just fixing it is probably the best solution. Thanks for the comment